README.md

```diff
@@ -3,6 +3,61 @@
 Introduction To Version Control With Git and GitHub
 ============================
 
+This is a repository to house materials for a 1/2 day workshop introducing participants to version control using git and the popular code hosting platform, GitHub.
+
+The goal of this workshop is to teach new-to-programming data professionals how to harness the power of version control by giving attendees a foundation in the fundamentals of version control and git.
```

Comment:
Not necessarily new to programming but likely newer to version control?

Reply:
Maybe add a url to git? e.g. [git](https://git-scm.com)